program/src/error.rs

```diff
@@ -0,0 +1,37 @@
+//! Program error types.
+
+use {
+    num_derive::FromPrimitive,
+    solana_program::{
+        decode_error::DecodeError,
+        msg,
+        program_error::{PrintProgramError, ProgramError},
+    },
+    thiserror::Error,
+};
+
+/// Errors that can be returned by the Config program.
+#[derive(Error, Clone, Debug, Eq, PartialEq, FromPrimitive)]
+pub enum AddressLookupTableError {
+    /// Instruction modified data of a read-only account.
+    #[error("Instruction modified data of a read-only account")]
+    ReadonlyDataModified = 10,
```

Comment:
Is there a reason this is `10`? The other PR didn't set a number

Reply:
Oops, accidentally removed my comment, but it's to avoid collisions with the custom errors from System program.

Reply:
Added the comment back!